### DISCUSSION TITLE
Comparison with workerize-loader?

### DISCUSSION THREAD

**Discussion by rauschma** [bitair-org/concurrent.js#4] [General]
If you have the time: In the readme, I’d find a comparison with [Workerize](https://github.com/developit/workerize) interesting.

**Comment by beshan** [comment]
Thank you for bringing it up. For some legal and technical reasons, we strictly avoid checking any similar libraries while designing ours. It's also a little soon for doing a comparison since we are still in the design phase and everything is subject to change. Our main goal is to avoid any stringly-typed syntax in our entire design, e..g:
`const task = new ParallelTask('./module_path.js', 'someFunction', ['arg1', true, 'arg3'])`

**Reply by rauschma** [reply]
Makes sense. Thanks for the reply!
